frontend/src/v5/ui/components/shared/sso/microsoftButton.component.tsx

```diff
@@ -31,7 +31,6 @@ export const MicrosoftButton = styled(Button).attrs({
 	width: fit-content;
 	font-weight: 500;
 	font-size: 12px;
-	border-radius: 0;
```

Comment:
Removing the border-radius here means the MS button has rounded corners. I am not sure we should do this, did George specifically asked for this?